mne/time_frequency/psd.py

```diff
@@ -11,9 +12,9 @@
 
 
 @verbose
-def compute_raw_psd(raw, tmin=0., tmax=np.inf, picks=None,
-                    fmin=0, fmax=np.inf, n_fft=2048, pad_to=None, n_overlap=0,
-                    n_jobs=1, plot=False, proj=False, verbose=None):
+def compute_raw_psd(raw, tmin=0., tmax=np.inf, fmin=0, fmax=np.inf,
```

Comment:
you should avoid changing parameters order. it will break people's code which means yours in 6 months ;) put back picks where is was and check other params.